File: CPP/Syntax/For.c
#include <stdio.h>
 
int main () {
    for (int x = 0; x < 10; x++) {
    	printf("%d\n", x);
    	if (x % 2 == 0) {
    		x++;
    		continue;
    	}
    }
}

//https://pt.stackoverflow.com/q/80589/101

File: JavaScript/Syntax/For.js
var objeto = [ 1, 2 ];
for (var chave in objeto) console.log(objeto[chave]);

for (var chave in objeto) if (objeto.hasOwnProperty(chave)) console.log(objeto[chave]);

objeto.forEach(function(valor, indice) {
    console.log(valor);
});

//https://pt.stackoverflow.com/q/90608/101
